taxi/forms.py

```diff
@@ -0,0 +1,52 @@
+import re
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+class DriverCreationForm(UserCreationForm):
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + (
+            "first_name", "last_name", "license_number",
+        )
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if not re.fullmatch(r"[A-Z]{3}\d{5}", license_number):
+            raise ValidationError(
+                "License number must consist "
+                "of 3 uppercase letters followed by 5 digits."
+            )
+        return license_number
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    class Meta:
+        model = Driver
+        fields = ("license_number",)
+
+    def clean_license_number(self):
+        license_number = self.cleaned_data["license_number"]
+        if not re.fullmatch(r"[A-Z]{3}\d{5}", license_number):
+            raise ValidationError(
+                "License number must consist of "
+                "3 uppercase letters followed by 5 digits."
+            )
+        return license_number
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
The queryset for the `drivers` field in `CarForm` should be filtered to include only `Driver` instances. Currently, it uses `get_user_model().objects.all()`, which retrieves all user instances. You should filter this queryset to include only those users who are drivers, possibly by using `Driver.objects.all()` or a similar filter.